src/makielayout/blocks/axis.jl

```diff
@@ -1198,8 +1196,10 @@ end
 Sets the space allocated for the yticklabels of the `Axis` to the minimum that is needed and returns that value.
 """
 function tight_yticklabel_spacing!(ax::Axis)
-    space = tight_ticklabel_spacing!(ax.yaxis)
-    return space
+    spacing = tight_ticklabel_spacing!(ax.yaxis)
+    # Needed to not reset to automatic... But still doesn't keep the value somehow?
```

Comment:
what are these commented out parts for?

Reply:
I started debugging `timed_ticklabelspace_reset` but couldn't figure out if it work as expected and what even expected means... setting ` ax.yticklabelspace = spacing` seems to help `timed_ticklabelspace_reset` to have the correct prev values, but still didn't seem to work correctly, so I reverted it, but left the comment to not loose the thought.